src/pages/home/report/ReportActionsView.tsx

```diff
@@ -347,7 +347,7 @@ function ReportActionsView({
         };
     }, [isTheFirstReportActionIsLinked]);
 
-    if (isLoadingInitialReportActions && visibleReportActions.length === 0 && !isOffline) {
```

Comment:
This change effectively removes the more eager rendering pattern that showed us offline-available content immediately.